pyqtgraph/graphicsItems/AxisItem.py

```diff
@@ -1112,7 +1110,7 @@ def generateDrawSpecs(self, p):
                 vstr = strings[j]
                 if vstr is None: ## this tick was ignored because it is out of bounds
                     continue
-                vstr = asUnicode(vstr)
+                vstr = str(vstr)
```

Comment:
As earlier, we already filter out the `None` case, leaving us with the `str` case, so we can remove this line altogether.